examples/server-side-rendering/src/client/App.js

```diff
@@ -30,6 +33,9 @@ const Moment = loadable.lib(() => import('moment'))
 
 const App = () => (
   <div>
+  <PrerenderedComponent live={Native.load()}>
```

Comment:
should we create one in loadable components, using PrerenderedComponent behid de scenes?¿